runtime/oti/VMHelpers.hpp

```diff
@@ -2109,6 +2109,40 @@ class VM_VMHelpers
 			currentThread->privateFlags &= ~(UDATA)J9_PRIVATE_FLAGS_VIRTUAL_THREAD_HIDDEN_FRAMES;
 		}
 	}
+
+	/**
+	 * Check if thread is in suspended state.
+	 *
+	 * Current thread must have VM access.
+	 *
+	 * @param[in] currentThread the current J9VMThread
+	 * @param[in] thread the target Thread object
+	 * @return true if thread is suspend, false otherwise.
+	 *
+	 */
```

Comment:
A article is missing in the summary, i.e. "the"
```
	 * Check if thread is in the suspended state.
```
In the `@return` clause, "suspend" should be "suspended". By convention, the clause does not end with a period, and line 2121 should be removed.
```
	 * @return true if thread is suspended, false otherwise
	 */
```